src/lnp/session/noise/chacha.rs

```diff
@@ -0,0 +1,129 @@
+// LNP/BP Core Library implementing LNPBP specifications & standards
+// Written in 2020 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
```

Comment:
Again, just replace my name with names of those who actually written the code

Reply:
Done